Guardfile

```diff
@@ -16,7 +16,7 @@ guard :rspec, cmd: "bin/rspec" do
   dsl.watch_spec_files_for(ruby.lib_files)
 
   # Rails files
-  rails = dsl.rails(view_extensions: %w(slim))
+  rails = dsl.rails(view_extensions: %w[slim])
```

Comment:
yep, looks like our friend @shir forced this change here https://github.com/bbatsov/rubocop/issues/4039